src/sql/base/browser/ui/table/plugins/headerFilter.plugin.ts

```diff
@@ -172,8 +174,20 @@ export class HeaderFilter<T extends Slick.SlickData> {
 	}
 
 	private showFilter(filterButton: HTMLElement): void {
+		const offset = jQuery(filterButton).offset();
+		let menuleft = offset.left - 226 /* filter width*/ + jQuery(filterButton).width() - 8;
```

Comment:
Probably easier to just make const values for these

Reply:
needs to be updated later for offscreen scenario, so have to leave it as `let`